cypress/integration/e2e/article.bannerPicker.e2e.spec.js

```diff
@@ -1,30 +0,0 @@
-/* eslint-disable no-undef */
-/* eslint-disable func-names */
```

Comment:
FYI @tjmw - doesn't look like it was doing much, but as you added it, you might have a plan for it (and can resurrect it from here 💀 )

Reply:
In fact, Amy and I totally stole @tjmw approach and used in [our PR](https://github.com/guardian/dotcom-rendering/pull/2091) and now I'm brutally removing Tom's tests before the other PR is even merged. Sorry for stealing your ideas _and_ thunder 😄 but thanks @tjmw for the doing all the work!

Reply:
![tenor-199409778](https://user-images.githubusercontent.com/638051/100597276-28557e80-32f5-11eb-9dd0-41bbdeae6db6.gif)

Reply:
Hey cool thanks for the heads-up! Glad the work was useful @oliverlloyd :-) I think @liywjl ended up commenting or removing some of the tests in here because at some point they'd started failing and we couldn't figure out the cause. Out of scope for your other PR but I'd love to resurrect the test that checks we invoke the contributions banner service. Previously I merged a change which accidentally stopped us from sending the request and it didn't get picked up until it was noticed in prod, so having something catch that on CI would be 👍  👍  I'd be happy to pair with someone on the dotcom team to figure it out.